drop-transfer/src/ws/client/mod.rs

```diff
@@ -223,8 +223,8 @@ impl RunContext<'_> {
         let (tx, rx) = mpsc::unbounded_channel();
 
         let mut lock = self.state.transfer_manager.lock().await;
-        lock.insert_transfer(self.xfer.clone(), TransferConnection::Client(tx))
-            .await?;
+        lock.insert_transfer(self.xfer.clone(), TransferConnection::Client(tx))?;
+        drop(lock);
```

Comment:
Is there a reason this can't be a one-liner?

Reply:
Probaably

Reply:
done

Reply:
I think you forgot to add the changes before pushing :sweat_smile: